### PR TITLE
Temporarily skip Template.MVC tests on Helix

### PR DESCRIPTION
**Pull request by captainsafia**
See https://github.com/dotnet/aspnetcore-internal/issues/4550.

cc: @BrennanConroy @SteveSandersonMS @wtgodbe

### CODE REVIEW

File: src/ProjectTemplates/test/Templates.Mvc.Tests/Templates.Mvc.Tests.csproj [hunk 1]
```diff
@@ -64,4 +64,13 @@
 
   <!-- Shared testing infrastructure for running E2E tests -->
   <Import Condition="'$(SkipTestBuild)' != 'true'" Project="..\..\TestInfrastructure\PrepareForTest.targets" />
+
+  <!-- See https://github.com/dotnet/aspnetcore/issues/55695 -->
```
Comment (wtgodbe) [inline src/ProjectTemplates/test/Templates.Mvc.Tests/Templates.Mvc.Tests.csproj:68]: If we just want to skip these on Helix, set `<BuildHelixPayload>false</BuildHelixPayload>`

**Review by wtgodbe — APPROVED** [review]

**Comment by amcasey** [comment]
I can't help but notice that the CI failures are MVC helix tests. 😆

**Comment by wtgodbe** [comment]
The failures are in `Mvc.FunctionalTests` - if it's the same root cause, we'd need to disable https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/test/Mvc.FunctionalTests/Microsoft.AspNetCore.Mvc.FunctionalTests.csproj as well

**Comment by captainsafia** [comment]
Closing these. These tests don't run against the PR build so skipping them will provide little value WRT to helping with PR flow.

**Pull request closed without merging**